DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -362,5 +368,58 @@ public void Get_section_content_returns_null_if_customisation_is_inactive()
             // Then
             result.Should().BeNull();
         }
+
+        [Test]
+        public void Get_section_content_can_have_consolidation_path()
+        {
+            // When
+            const int customisationId = 1499;
+            const int candidateId = 6;
+            const int sectionId = 74;
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            // Then
+            const string expectedConsolidationPath = "https://www.dls.nhs.uk/tracking/MOST/Word07Core/cons/WC07-Exercise_1.zip";
+
+            result.ConsolidationPath.Should().Be(expectedConsolidationPath);
+        }
+
+        [Test]
+        public void Get_section_content_can_have_no_consolidation_path_but_still_return()
+        {
+            // When
```

Comment:
This could do with being split into given/when/then. I'd put the inputs and expected values in the given section 🙂